### PR TITLE
new dockers files that compiles the pktfwr both arm and intel

### PR DESCRIPTION
**Pull request by ronniesa**
enabling function publishing from vscode

### CODE REVIEW

File: LoRaEngine/modules/LoRaWanNetworkSrvModule/module.json [hunk 1]
```diff
@@ -4,12 +4,11 @@
   "image": {
     "repository": "$CONTAINER_REGISTRY_ADDRESS/lorawannetworksrvmodule",
     "tag": {
-      "version": "0.1.1",
+      "version": "0.2.0",
```
Comment (kaizimmerm) [inline LoRaEngine/modules/LoRaWanNetworkSrvModule/module.json]: Shouldn't we try to keep that one on the same version as overall release, i.e. "0.1.0"

Comment (ronniesa) [reply LoRaEngine/modules/LoRaWanNetworkSrvModule/module.json]: problem is that I have used the 0.1.xxx so starting from a lower number of what I have in the gateways and registry needs clean up. I found it easy to have 0.2.xxx

Comment (kaizimmerm) [reply LoRaEngine/modules/LoRaWanNetworkSrvModule/module.json]: `LoraKeysManagerFacade/.vscode` I understand, are the others needed as well?

Comment (ronniesa) [reply LoRaEngine/modules/LoRaWanNetworkSrvModule/module.json]: fixed .vscode

**Pull request merged**